silk/views/sql.py

```diff
@@ -20,10 +22,16 @@ def get(self, request, *_, **kwargs):
             'request': request,
         }
         if request_id:
+            duplicate_queries = defaultdict(lambda: -1)
             silk_request = Request.objects.get(id=request_id)
             query_set = SQLQuery.objects.filter(request=silk_request).order_by('-start_time')
             for q in query_set:
                 q.start_time_relative = q.start_time - silk_request.start_time
+                duplicate_queries[q.query_structure] += 1
+            structures = list(duplicate_queries.keys())
+            for q in query_set:
+                q.num_duplicates = duplicate_queries[q.query_structure]
+                q.duplicate_id = structures.index(q.query_structure)
```

Comment:
Queries/requests created before applying migration won't have a structure
```suggestion
            for q in query_set:
                if q.query_structure:
                    q.num_duplicates = duplicate_queries[q.query_structure]
                    q.duplicate_id = structures.index(q.query_structure)
```